pontoon/administration/forms.py

```diff
@@ -12,6 +13,7 @@ def label_from_instance(self, obj):
 
 class ProjectForm(forms.ModelForm):
     contact = ContactChoiceField(None, required=False)
+    info = HtmlField()
```

Comment:
`info = HtmlField(widget=forms.Textarea)`

It would be even better if you could move the widget definition into the field itself, because we will always want to use textarea instead of the input field.